src/progress_bar_exercise/components/Button/Button.scss

```diff
@@ -0,0 +1,34 @@
+@import "../../../shared_styles/resets.scss";
+@import "../../../shared_styles/colors.scss";
+@import "../../../shared_styles/typography.scss";
+@import "../../../shared_styles/general.scss";
+
+$vertical-padding: 18px;
+$horizontal-padding: 33px;
+
+button.btn {
+  border-width: 1px;
+  border-style: solid;
+
+  text-transform: uppercase;
+  border-radius: 30px;
+  padding: $vertical-padding $horizontal-padding;
+
+  &.primary {
+    border-color: $green;
+    color: $green;
+  }
+  &.danger {
+    border-color: $red;
+    color: $red;
+  }
+
+  &:hover {
+    border-width: 2px;
+    padding: $vertical-padding - 1 $horizontal-padding - 1;
```

Comment:
**Note for reviewers:**

Removing 1 since it's the difference in `border-width` compared with the normal state. Same happens with the active state.